### ISSUE TITLE
real-time translations of text and audio should be in scope for this document  

### ISSUE THREAD

**Issue by jasonjgw** [w3c/apa#252]
This issue was raised at TPAC 2021.

**Comment by sehollier** [comment]
My understanding from the TPAC comment is that this relates to WCAG guideline 3.1 in that translation functionality. I think the issue is that their isn't support for language translation to be identified correctly for assistive technologies.

**Comment by jasonjgw** [comment]
If text is being translated and written to an element on a Web page, then an HTML "lang" attribute can be used to identify the language.

As Scott notes, this is already addressed by WCAG - and we advise meeting software implementers to follow WCAG - so I am not sure that we should write about this issue specifically.

**Comment by RealJoshue108** [comment]
@jasonjgw is right IMO - this is already covered in WCAG and not specific to this document. It may be useful to refer to the related SC etc

**Comment by sehollier** [comment]
Agree with above comments, closing issue.
